.github/workflows/update-from-protocol-contracts-solana.yaml

```
@@ -0,0 +1,45 @@
+          path: protocol-contracts-solana
+
+      - name: Update docs
+        run: |
+          mkdir -p src/pages/developers/protocol/
+          rm -f src/pages/developers/protocol/solana.md
+          cp protocol-contracts-solana/docs/gateway.md src/pages/developers/protocol/solana.md
+          rm -rf protocol-contracts-solana
+
+      - name: Check for changes
+        id: check-changes
+        run: |
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)